### ISSUE TITLE
Env variable no longer readable when config:cache is run

### ISSUE THREAD

**Issue by salhdev** [encodia/laravel-health-env-vars#7]
Hi,
I ran across the problem when staging the laravel app where we run the config:cache like it would be in production and then all my .env I was checking for went missing. After a bit of looking around I found out where it was coming from.
[Here is an interesting article I found ](https://andy-carter.com/blog/env-gotcha-in-laravel-when-caching-configuration)
And here is the extract from Laravel docs : 
`If you execute the config:cache command during your deployment process, you should be sure that you are only calling the env function from within your configuration files. Once the configuration has been cached, the .env file will not be loaded and all calls to the env function will return null.`

So in conclusion I believe it would save others a lot of time looking if in the Readme this would be mentioned.

Thanks

**Comment by eleftrik** [comment]
Hi @salhdev, I didn't mention it in the documentation because it's the standard behavior of .env variables after the Laravel config has been cached. 
That being said, adding it to the documentation of this package might be useful.
Thanks for the suggestion.

Erik

**Comment by encodiaweb** [comment]
@salhdev Updated README.md in versione `1.3.0`
